controllers/ovncontroller_controller.go

```diff
@@ -521,6 +551,63 @@ func (r *OVNControllerReconciler) generateServiceConfigMaps(
 	return configmap.EnsureConfigMaps(ctx, h, instance, cms, envVars)
 }
 
+// generateExternalConfigMaps - create configmaps for external dataplane consumption
+func (r *OVNControllerReconciler) generateExternalConfigMaps(
+	ctx context.Context,
+	h *helper.Helper,
+	instance *v1beta1.OVNController,
+	sbCluster *v1beta1.OVNDBCluster,
+	envVars *map[string]env.Setter,
+) error {
+	// Create/update configmaps from templates
+	cmLabels := labels.GetLabels(instance, labels.GetGroupLabel(ovncontroller.ServiceName), map[string]string{})
+
+	externalEndpoint, err := sbCluster.GetExternalEndpoint()
+	if err != nil {
+		return err
+	}
+
+	externalTemplateParameters := make(map[string]interface{})
+	externalTemplateParameters["OvnRemote"] = externalEndpoint
+	externalTemplateParameters["OvnEncapType"] = instance.Spec.ExternalIDS.OvnEncapType
+	externalTemplateParameters["OvnOpenFlowProbeInterval"] = strconv.Itoa(int(sbCluster.Spec.InactivityProbe))
+	externalTemplateParameters["OvnRemoteProbeInterval"] = strconv.Itoa(int(sbCluster.Spec.ProbeIntervalToActive))
+
+	cms := []util.Template{
+		// EDP ConfigMap
+		{
+			Name:          fmt.Sprintf("%s-config", instance.Name),
+			Namespace:     instance.Namespace,
+			Type:          util.TemplateTypeConfig,
+			InstanceType:  instance.Kind,
+			Labels:        cmLabels,
+			ConfigOptions: externalTemplateParameters,
+		},
+	}
+	return configmap.EnsureConfigMaps(ctx, h, instance, cms, envVars)
```

Comment:
Is there sensitive data (e.g. passwords) in this config map? In nova the transport_url contain the rabbitmq password so we decided to generate a Secret instead of the ConfigMap.

Reply:
No. There is no password authentication for OVN databases. As long as you can get to the port, you can do whatever you want.

Reply:
OK. Then it is a non issue for OVN